scripts/imagebuild.sh

```diff
@@ -53,7 +54,9 @@ docker build ${dockerfile_dir} -t ${REGISTRY}${REPO}/federation-v2:${TAG}
 echo "Pushing image with tag '${TAG}'."
 docker push ${REGISTRY}${REPO}/federation-v2:${TAG}
 
-if ! [ -z "${LATEST}" ]; then
+if [ "$LATEST" = "latest" ]; then
```

Comment:
The `=` operator isn't how the rest of the equality comparison tests in this file are written - would you use `==` so they're consistent?